### PR TITLE
bitcoin-core: Add `desc`

### PR DESCRIPTION
**Pull request by mavenor**
- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask {{cask_file}}` is error-free.
- [x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:
*(irrelevant)*

---
Add a generic description for “Bitcoin Core”

### CODE REVIEW

**Comment by EccentricHyphus37** [comment]
Ugh :/
Forgot to rebase first, so now my commit has the older `sha256` and `version` stanzas 🤦‍♂️

**Review by carlocab — DISMISSED** [review]

**Comment by carlocab** [comment]
Looks like your rebase still picked up some extraneous changes. (And merge commits, which BrewTestBot does not like -- hence the automerge-skip label.) Cleaned that up for you.

File: Casks/bitcoin-core.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
   url "https://bitcoincore.org/bin/bitcoin-core-#{version}/bitcoin-#{version}-osx.dmg"
   name "Bitcoin Core"
+  desc "Open-source, \"full\" Bitcoin client and wallet"
```
Comment (carlocab) [inline Casks/bitcoin-core.rb:7]:
```suggestion
  desc "Bitcoin client and wallet"
```
Better to keep the description as minimal as possible, I think. (You'll need to do `git pull --force` to pick up the changes from my rebase.)

Comment (EccentricHyphus37) [reply Casks/bitcoin-core.rb]: Yep, makes sense!

**Review by miccal — APPROVED** [review]

**Pull request merged**